app/src/main/java/com/example/keniphhomemaintenance/Navigation.kt

```diff
@@ -0,0 +1,41 @@
+package com.example.keniphhomemaintenance
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.navigation.NavType
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import androidx.navigation.navArgument
+import com.example.keniphhomemaintenance.dwellings.DwellingsScreenView
+import com.example.keniphhomemaintenance.ui.Screen
+
+@ExperimentalComposeUiApi
+@Composable
+fun Navigation() {
+    val navController = rememberNavController()
+    NavHost(navController = navController, startDestination = Screen.DwellingsScreen.route) {
+        composable(route = Screen.DwellingsScreen.route) {
+            DwellingsScreenView(navController = navController)
+        }
+        composable(
+            route = Screen.UtilitiesScreen.route + "/{name}" + "/{address}",
+            arguments = listOf(
+                navArgument("name") {
+                    type = NavType.StringType
+                    defaultValue = "Default Name"
+                    nullable = true
+                }, navArgument("address") {
+                    type = NavType.StringType
+                    defaultValue = "Default Address"
+                    nullable = true
+                }
+            )
+        ) { entry ->
+            UtilitiesScreen(
+                name = entry.arguments?.getString("name"),
```

Comment:
Using static constants for the names of the parameters ("name" and "address") would be great

Reply:
kept "name" and "address" as static, moved default values to strings.xml

Reply:
static constants being variables in a companion object:

    companion object {
        const val NAME_ARGUMENT_THING = "name" 
    }

Reply:
class NavArgParams {
    companion object {
        const val DWELLING_NAME = "name"
        const val DWELLING_ADDRESS = "address"
    }
}